commands/xinfo-consumers.md

```diff
@@ -1,29 +1,50 @@
-This command returns the list of consumers that belong to the `<groupname>` consumer group of the stream stored at `<key>`.
+Returns the list of consumers that belong to the `groupname` consumer group of the stream stored at `key`.
 
-The following information is provided for each consumer in the group:
+[Examples](#examples)
 
-* **name**: the consumer's name
-* **pending**: the number of pending messages for the client, which are messages that were delivered but are yet to be acknowledged
-* **idle**: the number of milliseconds that have passed since the consumer last interacted with the server
+## Required parameters
 
-@reply
+<details open>
+<summary><code>key</code></summary>
 
-@array-reply: a list of consumers.
+is name of the stream.
+</details>
 
-@examples
+<details open>
+<summary><code>groupname</code></summary>
+
+is name of the consumer group.
+</details>
+
+## Return
+
+XINFO CONSUMERS returns an @array-reply of consumers, with these elements:
+
+- `name` - Consumer's name
+- `pending` - Number of pending messages for the client, which are messages that were delivered but are yet to be acknowledged.
+- `idle` - Number of milliseconds that have passed since the consumer last interacted with the server and read an entry from the stream.
+
+## Examples
+
+<details open>
+<summary><b>Return the list of consumers</b></summary>
+
+Return the list of consumers from `mygroup`.
 
 ```
-> XINFO CONSUMERS mystream mygroup
+127.0.0.1:6379> XINFO CONSUMERS mystream mygroup
 1) 1) name
    2) "Alice"
    3) pending
    4) (integer) 1
    5) idle
-   6) (integer) 9104628
-2) 1) name
-   2) "Bob"
-   3) pending
-   4) (integer) 1
-   5) idle
    6) (integer) 83841983
 ```
+
+## See also
+
+`XINFO` | `XINFO GROUPS` | `XINFO STREAM` | `XINFO HELP`
```

Comment:
1. I'd rather add these to the frontmatter (e.g. as a `related` property) and not as text in the MD part
2. We need a definition of what we want to put in related commands - these aren't. `XINFO` isn't a real command, only a container (i.e. you can't call it by itself). `XINFO HELP` doesn't need to be listed IMO. The other two are related only in the sense of having the same container command, but that's about it.